.github/workflows/release.yaml

```diff
@@ -94,3 +94,24 @@ jobs:
         if: env.SHOULD_DEPLOY_DOCS == 'true'
         id: deployment
         uses: actions/deploy-pages@v1
+
+      - name: Checkout API Docs
+        if: env.SHOULD_DEPLOY_DOCS == 'true'
+        uses: actions/checkout@v3
+        with:
+          ref: docs
+        
+      - name: Generate and Push API Docs
+        if: env.SHOULD_DEPLOY_DOCS == 'true'
+        run: |
+          git pull origin master --no-edit
+          pnpm install
+          pnpm build
+          rm -f apps/docs/.gitignore
+          git add apps/docs/src/api/
+          git add apps/docs/.typedoc/api-links.json
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git config user.name "github-actions[bot]"
+          git commit -m "docs: API docs"
```

Comment:
Could we have the version number on the commit message here?

- "docs: API docs - `v0.50.1`"

Let's say we want — _in the future, at some point_ — to make all versions available with a dropdown for users to select which version they're using.

Reply:
Good idea

Reply:
I'll add to a follow-up issue